src/components/AnnotationBrowser/IndexConversionDialog.vue

```diff
@@ -0,0 +1,269 @@
+          contains three columns:
+          <ul class="mt-2">
+            <li><strong>UI Index</strong>: 1-based index shown in the UI</li>
+            <li>
+              <strong>JSON Index</strong>: 0-based index used in JSON exports
+            </li>
+            <li>
+              <strong>Label</strong>: String label from metadata (e.g., "5 min",
+              "H10", "2 µm")
+            </li>
+          </ul>
+        </v-alert>
+
+        <template v-if="!dataset">
+          <v-alert type="warning" text>
+            No dataset loaded. Please select a dataset first.
+          </v-alert>
+        </template>
+
+        <template v-else>
+          <!-- XY Dimension -->
+          <v-card outlined class="mb-3" v-if="hasXYDimension">
+            <v-card-title class="text-h6">XY Positions</v-card-title>
+            <v-card-text>
+              <div class="d-flex align-center">
+                <div class="flex-grow-1">
+                  <div>{{ xyCount }} positions</div>
+                  <div v-if="!hasXYLabels" class="text--secondary caption">
+                    No labels detected; index only
+                  </div>
+                </div>
+                <v-btn color="primary" @click="downloadXY">
+                  <v-icon left>mdi-download</v-icon>
+                  Download XY CSV
+                </v-btn>
+              </div>
+            </v-card-text>
+          </v-card>
+
+          <!-- Z Dimension -->
+          <v-card outlined class="mb-3" v-if="hasZDimension">
+            <v-card-title class="text-h6">Z Slices</v-card-title>
+            <v-card-text>
+              <div class="d-flex align-center">
+                <div class="flex-grow-1">
+                  <div>{{ zCount }} slices</div>
+                  <div v-if="!hasZLabels" class="text--secondary caption">
+                    No labels detected; index only
+                  </div>
+                </div>
+                <v-btn color="primary" @click="downloadZ">
+                  <v-icon left>mdi-download</v-icon>
+                  Download Z CSV
+                </v-btn>
+              </div>
+            </v-card-text>
+          </v-card>
+
+          <!-- Time Dimension -->
+          <v-card outlined class="mb-3" v-if="hasTimeDimension">
+            <v-card-title class="text-h6">Time Points</v-card-title>
+            <v-card-text>
+              <div class="d-flex align-center">
+                <div class="flex-grow-1">
+                  <div>{{ timeCount }} time points</div>
+                  <div v-if="!hasTimeLabels" class="text--secondary caption">
+                    No labels detected; index only
+                  </div>
+                </div>
+                <v-btn color="primary" @click="downloadTime">
+                  <v-icon left>mdi-download</v-icon>
+                  Download Time CSV
+                </v-btn>
+              </div>
+            </v-card-text>
+          </v-card>
```

Comment:
Please factorize these 3 sections into a new component

Reply:
I could… but it seems like a lot of overhead and obfuscation to make a new component for just this little bit of repetition, no?

Reply:
Sounds good